### ISSUE TITLE
Document all of usethis's options in one place

### ISSUE THREAD

**Issue by jennybc** [r-lib/usethis#1768]
usethis needs a help topic like these:

* https://gargle.r-lib.org/reference/gargle_options.html
* https://tibble.tidyverse.org/reference/tibble_options.html

(Some of) the options to document there:

* `usethis.destdir`
* `usethis.description`
* `usethis.protocol`
* `usethis.quiet`
* `usethis.overwrite`

**Comment by ateucher** [comment]
I have found in a few places a reference to `usethis.full_name` (including in [R Packages](https://github.com/hadley/r-pkgs/blob/4088c6e8d47c17138d8000fb299390c86fc74274/setup.Rmd#L82) and [this usethis article](https://usethis.r-lib.org/articles/usethis-setup.html#store-default-values-for-description-fields-and-other-preferences)), but I can't actually find anywhere where that option is actually consulted...

**Comment by jennybc** [comment]
Ah, this is leftover from previous versions of the license helpers:

```
~/rrr/usethis % git log -S usethis.full_name
commit 47e7bbe4bd12fba1aac13a2c4e086eb40b1203ce
Author: Hadley Wickham <h.wickham@gmail.com>
Date:   Thu Sep 10 12:35:32 2020 -0500

    Licensing improvements (#1208)
    
    * Make it easier to license non-packages
    * Don't change licenses that aren't templates
    * Change default copyright holder for MIT license
    * Clarify argument names
    * GPL and Apache licenses get version arguments
    * Update/simplify tests
    
    Fixes #1198. Fixes #1207. Fixes #1206.
```

So yeah I think from 47e7bbe on, there is no usage of `usethis.full_name`. Will you also open an issue or PRs on R Packages?
